db.go

```diff
@@ -891,7 +891,9 @@ func (db *DB) managed(writable bool, fn func(tx *Tx) error) (err error) {
 		}
 
 		errRollback := tx.Rollback()
-		err = fmt.Errorf("%v. Rollback err: %v", err, errRollback)
+		if errRollback != nil {
```

Comment:
It may be more elegant to call `tx.Rollback()` in `if`, like:
```go
if errRollback := tx.Rollback(); ...
```

Reply:
True :+1: